### PR TITLE
update TOC

### PR DESCRIPTION
**Pull request by isabelizimm**
## Related Issues and Dependencies

adding `pre-requisite.md` and `flask-application.md` to TOC

### CODE REVIEW

**Review by oindrillac — COMMENTED** [review]
/lgtm

File: _toc.yml [hunk 1]
```diff
@@ -18,5 +19,6 @@ parts:
   - caption: DevOps Lifecycle
     chapters:
     - file: docs/source/deploy-model
+    - file: docs/source/model-deployment/flask-application
```
Comment (chauhankaranraj) [inline _toc.yml:22]:
```suggestion
    - file: docs/source/deploy-model
      sections:
        - file: docs/source/model-deployment/flask-application
```
What if we put the deploy-as-a-flask-application page as a subsection of deploy-your-models? To explicitly show that it's just _one_ of the methods of  model deployment.

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]

File: _toc.yml (hunk 1 above)
Comment (isabelizimm) [reply _toc.yml:22]: sounds good! this gives space to add more methods, if wanted :) i like it

**Comment by isabelizimm** [comment]
(testing out github UI review tab to commit suggestions...if only it would rebase too!)

**Pull request merged**